src/arch/MovieTexture/MovieTexture_FFMpeg.cpp

```diff
@@ -442,32 +510,41 @@ int MovieDecoder_FFMpeg::DecodePacket( float fTargetTime )
 	return 0; /* packet done */
 }
 
-void MovieDecoder_FFMpeg::GetFrame( RageSurface *pSurface )
+bool MovieDecoder_FFMpeg::GetFrame(RageSurface* pSurface)
 {
 	avcodec::AVFrame pict;
-	pict.data[0] = (unsigned char *) pSurface->pixels;
+	pict.data[0] = (unsigned char*)pSurface->pixels;
```

Comment:
Is char* actually a char here or is it used to represent 8 bytes?

Reply:
This is preexisting, but it looks like it's converting an unsigned 8bit int, which funny enough is the exact type the reference has it needing

https://github.com/itgmania/itgmania/blob/5d4f9dcb07493ed4c51d6be23e9b41978162305a/src/RageSurface.h#L80
https://www.ffmpeg.org/doxygen/3.3/structAVFrame.html

I'll mess around with this later, but the conversion is maybe extraneous.